src/topology/uniform_space/uniform_convergence.lean

```diff
@@ -251,6 +306,40 @@ begin
   exact ⟨F m x, ⟨hm', htsymm (hm x hx)⟩⟩,
 end
 
+lemma uniform_cauchy_seq_on.mono {s' : set α} (hf : uniform_cauchy_seq_on F p s) (hss' : s' ⊆ s) :
+  uniform_cauchy_seq_on F p s' :=
+λ u hu, (hf u hu).mono (λ x hx y hy, hx y (hss' hy))
+
+/-- Composing on the right by a function preserves uniform convergence -/
+lemma uniform_cauchy_seq_on.comp {γ : Type*} (hf : uniform_cauchy_seq_on F p s) (g : γ → α) :
+  uniform_cauchy_seq_on (λ n, F n ∘ g) p (g ⁻¹' s) :=
+λ u hu, (hf u hu).mono (λ x hx y hy, hx (g y) hy)
+
+/-- Composing on the left by a uniformly continuous function preserves
+uniform convergence -/
+lemma uniform_cauchy_seq_on.comp' [uniform_space γ] {g : β → γ} (hf : uniform_cauchy_seq_on F p s)
```

Comment:
```suggestion
lemma uniform_continuous.comp_uniform_cauchy_seq_on [uniform_space γ] {g : β → γ} (hf : uniform_cauchy_seq_on F p s)
```
also reorder the two explicit arguments

Reply:
This is structured in parallel to `tendsto_uniformly_on.comp'` further up the file. Should this be changed?

Reply:
Easy enough to change. Building now to see if that causes downstream impacts.